control/init.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash
+if ! hash virtualenvwrapper.sh 2>/dev/null; then
+    echo "Please install virtualenvwrapper and make sure it is in your PATH"
+    echo ""
+    echo "  sudo apt-get install git python-dev python-pip"
+    echo "  sudo pip install virtualenv virtualenvwrapper"
+    return 1
+fi
+
+source virtualenvwrapper.sh
+
+if [ ! -d ~/.virtualenvs/ansible ]; then
+    echo "Creating ansible virtualenv..."
+    mkvirtualenv ansible
+    pip install -r ~/commcarehq-ansible/ansible/requirements.txt
+else
+    workon ansible
+fi
+
+if ! [ -L ~/commcarehq-ansible/config -o -d ~/commcarehq-ansible/config ]; then
+    # TODO setup permissions (add user to group?) to allow write to config repo
+    #su ansible -c "usermod -a -G ansible $USER"
+    ln -s /home/ansible/commcarehq-ansible/config ~/commcarehq-ansible/config
```

Comment:
Implemented ❤️ in https://github.com/dimagi/commcarehq-ansible/pull/467/commits/e143f119bf7d4e754e743b96fb478e241feb36fb